app/models/lesson.rb

```diff
@@ -20,6 +20,11 @@ class Lesson < ApplicationRecord
 
   attribute :completed, :boolean, default: false
 
+  def new?
```

Comment:
We'll need a unit test for this method.

Something very similar to what we've got for the like `created_today` scope should do the job:
https://github.com/TheOdinProject/theodinproject/blob/main/spec/models/like_spec.rb#L12

We'll just need to test the edges
- Created today
- Just within the new period - 1.month.ago
- Just outside of the new period - 1.month.ago + 1.day